converter/biostudies.py

```diff
@@ -105,12 +102,29 @@ def convert(self, hca_project: dict, additional_attributes: dict = None) -> dict
             'section': self.project_spec_section
             })
 
+        self.add_release_date(converted_project, hca_project)
+
         project_content = hca_project['content'] if 'content' in hca_project else None
         if project_content:
             self.__add_subsections_to_project(converted_project, project_content)
 
         return converted_project
 
+    @staticmethod
+    def add_release_date(converted_project, hca_project):
+        release_date = hca_project.get('releaseDate')
+        if release_date:
+            converted_project.get('attributes').append(
+                {
+                    'name': 'ReleaseDate',
+                    'value': release_date
+                }
+            )
+
+    @staticmethod
+    def __set_release_date(hca_project: dict):
+        hca_project.update({'releaseDate': datetime.now().strftime("%Y-%m-%dT%H:%M:%SZ")})
```

Comment:
Where is this format coming from? It is helpful to have format patterns as constants with a clear purpose and/or source, for example, ENA_DATA_FORMAT

Reply:
Actually, as I rewrote this part of the code this function is not even used, so I am going to remove it in the next PR. Tx